pyproject.toml

```diff
@@ -14,6 +14,12 @@ dependencies = [
   "evo-data-converters-vtk",
 ]
 
+[project.urls]
+Source = "https://github.com/SeequentEvo/evo-data-converters"
+Tracker = "https://github.com/SeequentEvo/evo-data-converters/issues"
+Homepage = "https://www.seequent.com/"
+Documentation = "https://developer.seequent.com/"
+
```

Comment:
Sorry, on closer inspection, you should add these to all of the `packages/*/pyproject.toml` files. This file doesn't get published to PyPI whereas those others do.